app/SearchApi/BcGov.Fams3.SearchApi.Contracts/PersonSearch/DataProviderEnums.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BcGov.Fams3.SearchApi.Contracts.PersonSearch
+{
+    public enum DataProviderEnums
+    {
```

Comment:
I think we should name it DataProviders (without the Enum)

Reply:
ok

Reply:
Cool.. Maybe just DataProvider (without the **s**)

Reply:
ok